forms-flow-web/src/components/Form/constants/FormTable.js

```diff
@@ -29,16 +31,15 @@ function FormTable() {
   const pageNo = useSelector((state) => state.bpmForms.page);
   const limit = useSelector((state) => state.bpmForms.limit);
   const totalForms = useSelector((state) => state.bpmForms.totalForms);
-  const sortOrder = useSelector((state) => state.bpmForms.sortOrder);
+  const formsort = useSelector((state) => state.bpmForms.sort);
   const searchFormLoading = useSelector(
     (state) => state.formCheckList.searchFormLoading
   );
-  const isAscending = sortOrder === "asc" ? true : false;
   const redirectUrl = MULTITENANCY_ENABLED ? `/tenant/${tenantKey}/` : "/";
   const isApplicationCountLoading = useSelector((state) => state.process.isApplicationCountLoading);
   const { createDesigns } = userRoles();
   const [expandedRowIndex, setExpandedRowIndex] = useState(null);
-  const [assending, setAssending] = useState(true);
+  const [formSort ,setFormSort] = useState(formsort);  
```

Comment:
line 34 and 42 have same variable name with cahnge in casing. Can you please change one of the variables as it would be difficult to manage lateron @fahad-aot

Reply:
Updated.